### DISCUSSION TITLE
Should we leave GitHub?

### DISCUSSION THREAD

**Discussion by smichel17** [PaperWM-community/PaperWM#30] [General]
I was going to post this over in #27, but I think it's a big enough question that it deserves its own topic, for more visibility.

https://sfconservancy.org/GiveUpGitHub/ has a good list of the reasons why.

Now is by far the easiest time to switch forges: before everyone has learned that this is the new location, and before there's a bunch of issues created and integrations with the platform. When there are still relatively few people who are involved in the discussion, so it's easier to reach consensus.

If we decide to move (which I am *personally* in favor of, but will not insist on), I think [CodeBerg](https://codeberg.org/) is probably the best option. I like that SourceHut is attempting to be something other than a GitHub clone, but I think the combination of it being more different and early in its development make it not the best choice at the moment, unless everyone involved here prefers its philosophy and email-based workflow.

**Comment by ccope** [comment]
IMO we should get people to migrate here from the other github repository first. Once people are tracking this fork then replacing it with a redirect elsewhere would mean the userbase is more likely to follow. I think keeping it here until there are more active maintainers is the best plan though, it will be harder to attract new maintainers on a less-centralized/popular platform.

**Reply by smichel17** [reply]
I agree it will be easier to attract new maintainers here.

I don't think I agree about the userbase being more likely to follow. Every transition loses some people; I'd err towards making fewer transitions. Although, I don't think it will make that much of a difference either way: PaperWM is going to break on people when they upgrade to Gnome 40, and then they'll go to the GitHub issues of the other repo, see the pinned issue about support, and follow the link to the new repo, whether it's here or elsewhere. If the original were still maintained or there were another fork, then I'd expect it would make more of a difference.

**Comment by wraithm** [comment]
IMHO, now that you have access to `paperwm/PaperWM`, we should keep it there.
